apis/compute/v1alpha2/network_types.go

```diff
@@ -195,37 +189,46 @@ type GCPNetworkStatus struct {
 	// An auto mode VPC network starts with one subnet per region. Each
 	// subnet has a predetermined range as described in Auto mode VPC
 	// network IP ranges.
-	AutoCreateSubnetworks bool `json:"autoCreateSubnetworks,omitempty"`
+	// +optional
+	AutoCreateSubnetworks *bool `json:"autoCreateSubnetworks,omitempty"`
 
 	// CreationTimestamp: Creation timestamp in RFC3339 text
 	// format.
-	CreationTimestamp string `json:"creationTimestamp,omitempty"`
+	// +optional
+	CreationTimestamp *string `json:"creationTimestamp,omitempty"`
 
 	// Description: An optional description of this resource. Provide this
 	// field when you create the resource.
-	Description string `json:"description,omitempty"`
+	// +optional
+	Description *string `json:"description,omitempty"`
 
 	// GatewayIPv4: The gateway address for default routing
 	// out of the network, selected by GCP.
-	GatewayIPv4 string `json:"gatewayIPv4,omitempty"`
+	// +optional
+	GatewayIPv4 *string `json:"gatewayIPv4,omitempty"`
 
 	// Id: The unique identifier for the resource. This
 	// identifier is defined by the server.
-	ID uint64 `json:"id,omitempty"`
+	// +optional
```

Comment:
https://github.com/crossplaneio/crossplane/blob/edd5243d64e36946df0e98facc2abac97f80569c/design/one-pager-managed-resource-api-design.md#pointer-types-and-markers